apps/api/src/employee/employee.service.ts

```diff
@@ -22,4 +24,48 @@ export class EmployeeService {
   async getUserByKeycloakId(keycloakId: string): Promise<EmployeeEntity | undefined> {
     return await this.employeeRepository.findOne({ keycloakId });
   }
+
+  /**
+   * List and filter employees,
+   * Only for administrator purposes
+   * We will build UI for this when we extend our scope to provide user-management
+   * @param name optional name wise filter
+   * @returns Employee/User's list
+   */
+  async getEmployeeList(name: string): Promise<EmployeeEntity[]> {
+    const query: FindManyOptions<EmployeeEntity> = {
+      select: ['id', 'name', 'role'],
+    };
+    if (name && name != '') {
+      query.where = { name: ILike(`%${name}%`) };
+    }
+    return await this.employeeRepository.find(query);
+  }
+
+  /**
+   * Update role of the provided employee list
+   * @param ids Employee Ids whose role we are updating
+   * @param role role
+   */
```

Comment:
Should we prevent an admin user from creating more admin users? I am just thinking about the risks of privilege escalation.